projects/noloan/app/src/main/java/hasadna/noloan/Messages.java

```diff
@@ -218,6 +230,26 @@ public void notifyListeners(int index, SmsMessage smsMessage, Type type) {
     }
   }
 
+  public FirebaseUser getFirebaseUser() {
+    return firebaseUser;
+  }
+
+  public List<SmsMessage> getDbMessages() {
+    return dbMessages;
+  }
+
+  public List<SmsMessage> getInboxMessages() {
+    return inboxMessages;
+  }
+
+  public void setInboxMessages(List<SmsMessage> inboxMessages) {
+    this.inboxMessages = inboxMessages;
+  }
+
+  public void setContext(Context context) {
```

Comment:
Where do you use `setContext()`?

Reply:
I wanted to move `readSmsFromDevice()` from the `MainActivity` to be part of `SmsMessages` functions, it was needed for that,but I ran into a problem. Not sure if that's correct though in terms of structure, I'll raise it up in the channel.
Anyway, removed it for now :+1: .